src/org/labkey/test/LabKeySiteWrapper.java

```diff
@@ -720,6 +722,29 @@ else if (getDriver().getTitle().startsWith("Sign In"))
                     // Note: leave the self-report setting unchanged
                     customizeSitePage.save();
                 }
+
+                //Waiting for search service to boot up
```

Comment:
We should include a note here about why this wait is being done:
https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=50601